pkg/filesystem/local_directory_darwin.go

```diff
@@ -17,3 +18,7 @@ type deviceNumber = int32
 func (d *localDirectory) Mknod(name path.Component, perm os.FileMode, dev int) error {
 	return status.Error(codes.Unimplemented, "Creation of device nodes is not supported on Darwin")
 }
+
+func clonefileImpl(oldFd int, oldName string, newFd int, newName string) error {
```

Comment:
s/oldFd/oldFD/; same for newFd.

Go's style guide suggests that abbreviations (HTTP, TCP, TLS) are always capitalized. We do it consistently in the Buildbarn tree, except when dealing with generated Protobuf field names.